lettuce/reporters.py

```diff
@@ -124,10 +125,16 @@ class ObservableReporter:
     >>> # simulation.reporters.append(reporter)
     """
 
-    def __init__(self, observable, interval=1, out=sys.stdout):
+    def __init__(self, observable, interval=1, out=[], print_to_screen=True, save_to_file=False, filename_base="./data/output"):
```

Comment:
If you want to write to a file and to the console, you don't need to change the reporter. You could either 
1. just use two reporters or 
2. define an `out` object that writes to the console and to a file.

For 2., I mean something like this (not tested):
```
class FileAndConsoleStream:
    def __init__(self, filename):
        self.file = open(filename, "w") 
    def write(self, *args, **kwargs):
        sys.stdout.write(*args, **kwargs)
        self.file.write(*args, **kwargs)
    def __del__(self):
        self.file.close()
        
reporter = ObservableReporter(observable, interval=10, out=FileAndConsoleStream("myfile.txt"))

Reply:
Haha, chapeau, this works right off the bat, wow! 😄